Lib/fontbakery/cli.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+import argparse
+import importlib
+import sys
+
+
+def main(args=None):
+    subcommands = {
```

Comment:
can we (maybe later) add an auto discovery mechanism to this. It would spare us from having to register each command. Also we have that kind of thing in `bin/fontbakery` so this would be a regression.

Reply:
Sure. I just need to look into discovering the submodules of a module :)